src/gui/print_widget.cpp

```diff
@@ -264,6 +267,10 @@ PrintWidget::PrintWidget(Map* map, MainWindow* main_window, MapView* main_view,
 	
 	overprinting_check = new QCheckBox(tr("Simulate overprinting"));
 	layout->addRow(overprinting_check);
+
+    world_file_check = new QCheckBox(tr("Save world file"));
+    layout->addRow(world_file_check);
+    world_file_check->hide();
```

Comment:
In my opinion, World File generation should not be offered in case map georeferencing is not set. Not sure if `isLocal()` is the right method to check georeferencing validity as `Georeferencing`'s internal state handling looks somewhat conflated to me.

```
diff --git a/src/gui/print_widget.cpp b/src/gui/print_widget.cpp
index ebdec3c758c3..4755a1654337 100644
--- a/src/gui/print_widget.cpp
+++ b/src/gui/print_widget.cpp
@@ -271,6 +271,8 @@ PrintWidget::PrintWidget(Map* map, MainWindow* main_window, MapView* main_view,
        world_file_check = new QCheckBox(tr("Save world file"));
        layout->addRow(world_file_check);
        world_file_check->hide();
+       if (map->getGeoreferencing().isLocal())
+               world_file_check->setEnabled(false);
        
        color_mode_combo = new QComboBox();
        color_mode_combo->setEditable(false);
```

... or shorter `setEnabled(!...isLocal());`.